control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -175,6 +175,8 @@ pub struct VolumeSpec {
     /// Affinity Group related information.
     #[serde(default)]
     pub affinity_group: Option<AffinityGroup>,
+    /// Number of snapshots taken on this volume.
+    pub num_snapshots: u32,
```

Comment:
```suggestion
    #[serde(default)]
    pub num_snapshots: u32,
```

Reply:
Why do we need this?

Reply:
Otherwise we would need to add the list of snapshots in grpc level?

Reply:
Added `#serde(skip)` as discussed.